export: edit page (#12573)
tldr-pages/tldr: pages/linux/export.md

- Append a pathname to the environment variable `PATH`:

`export PATH=$PATH:{{path/to/append}}`

> - Display a list of all exported variables and functions:

Reviewer: ```suggestion
- Display all exported variables and functions in a executable format:
```
How about this?

Author: TBH I just copied it from `export --help` so I'm not _super_ picky about the specific message. However, I'm a bit of a noob, so the "list"  part makes sense to me where the "executable" part is honestly a bit confusing... but my Bash skills are hardly up to par.

Reviewer 2: I'm inclined to agree with the `list of` version.

The current version is much more readable, I vote to keep as is.

Reviewer 3: I actually prefer @vitorhcl's suggestion. It's a more accurate description because it indicates to the user that you can use the output to reproduce the environment. Also, <s>a</s> an.

Reviewer 2: Its been a while since I contributed, aren't these pages meant to be accessible to everyone, to a new user, what is an `executable format`?

Author: @arcanemachine  This is my feeling as well TBH.

Reviewer: Yeah, I agree that the wording is a bit confusing, I accept a better one. But I think it is important to leave this clear in some way.

Reviewer 3: > to a new user, what is an `executable format`?

Hmm fair.

> - Display a list of all exported variables and functions:

Reviewer 2: ```suggestion
- Display a list of in-use exported variables in shell command form.
```

> - Display a list of all exported variables and functions:

Reviewer 3: I like the "shell command form" expression. But maybe "active" is a more descriptive word than "in-use" (in use by what?).

```suggestion
- Display a list of active exported variables in shell command form.
```

> - Display a list of all exported variables and functions:

Reviewer 2: @arcanemachine Could you apply a change, then we can merge.

> - Display a list of all exported variables and functions:

Author: @CleanMachine1 Done :)
